### PR TITLE
Fix most of the broken links.

### PR DESCRIPTION
**Pull request by markstory**
There were a few broken links that I couldn't figure out a sane target for.

Refs #3735

### CODE REVIEW

File: en/appendices/new-features-in-cakephp-1-3.rst [hunk 1]
```diff
@@ -114,7 +114,7 @@ Logging
 
 Logging and ``CakeLog`` have been enhanced considerably, both in
 features and flexibility. See
-`New Logging features <http://book.cakephp.org/view/1194/Logging>`_ for more information.
+`New Logging features <http://book.cakephp.org/1.3/en/The-Manual/Common-Tasks-With-CakePHP/Logging.html/>`_ for more information.
```
Comment (ravage84) [inline en/appendices/new-features-in-cakephp-1-3.rst]: This gives me a ``502 Bad Gateway``

File: es/installation/advanced-installation.rst [hunk 2]
```diff
@@ -23,7 +23,7 @@ con una estructura de directorios diferente.
 
 -  /app/webroot/index.php
 -  /app/webroot/test.php (si usas tests
-   `Testing <view/1196/Testing>`_ .)
+   `Testing </development/testing>`_ .)
```
Comment (ravage84) [inline es/installation/advanced-installation.rst]: Does the leading slash in the URL take the language into account?

**Comment by ravage84** [comment]
Thanks for your work! :beer:

File: en/appendices/new-features-in-cakephp-1-3.rst (hunk 1 above)
Comment (markstory) [inline en/appendices/new-features-in-cakephp-1-3.rst]: Derp trailing `/`.

File: es/installation/advanced-installation.rst (hunk 2 above)
Comment (markstory) [inline es/installation/advanced-installation.rst]: Yes.

File: en/appendices/new-features-in-cakephp-1-3.rst [hunk 3]
```diff
@@ -181,8 +181,7 @@ could do::
 
 -  Missing Validation methods now trigger errors, making debugging
    why validation isn't working easier.
--  Models now support
-   `virtual fields <http://book.cakephp.org/view/1608/Virtual-fields>`_
+-  Models now support virtual fields.
```
Comment (jadb) [inline en/appendices/new-features-in-cakephp-1-3.rst]: Should this link to http://book.cakephp.org/3.0/en/orm/entities.html#creating-virtual-fields

Comment (markstory) [inline en/appendices/new-features-in-cakephp-1-3.rst]: No because this is in the 1.3 new feature guide.

Comment (jadb) [inline en/appendices/new-features-in-cakephp-1-3.rst]: :dizzy_face:

**Pull request merged**